src/qt/combobox.cpp

```diff
@@ -296,15 +296,21 @@ void wxComboBox::SetSelection( long from, long to )
 
 void wxComboBox::SetInsertionPoint( long pos )
 {
-    // check if pos indicates end of text:
-    if ( pos == -1 )
-        m_qtComboBox->lineEdit()->end( false );
-    else
-        m_qtComboBox->lineEdit()->setCursorPosition( pos );
+    if ( m_qtComboBox->lineEdit() != nullptr )
```

Comment:
This is minor, of course, but I'd write it like

```suggestion
    if ( auto edit = m_qtComboBox->lineEdit() )
```

and then use `edit` inside the `if` to avoid calling `lineEdit()` thrice.

Reply:
I tried to match style to the rest of the file, e.g. `if ( m_qtComboBox->lineEdit() != nullptr )` already existed in the file.
From a quick search, there's only 4 usages of `if ( auto` in wx code, that's why I didn't prefer it.

Reply:
`auto` can only be used in the code added in 3.1, as we didn't use C++11 before, so this explains why it's relatively rare. But my main point wasn't even about using it, but rather about defining a variable instead of having multiple calls to `lineEdit()`. Again, this is pretty minor, but the code seems more readable to me if we don't have multiple repeated calls to the same function.

Reply:
I used `QLineEdit* const lineEdit` that has been used in other places, let me know if it's ok.